app/components/preprint-status-banner/component.js

```diff
@@ -0,0 +1,119 @@
+import Ember from 'ember';
+const {
+    computed,
+} = Ember;
+
+const messagePendingPre = 'components.preprint-status-banner.message.pending_pre';
+const messagePendingPost ='components.preprint-status-banner.message.pending_post';
+const messageAccepted = 'components.preprint-status-banner.message.accepted';
+const messageRejected = 'components.preprint-status-banner.message.rejected';
+
+const statusPending = 'components.preprint-status-banner.pending';
+const statusAccepted = 'components.preprint-status-banner.accepted';
+const statusRejected = 'components.preprint-status-banner.rejected';
+
+const workflowPre = 'components.preprint-status-banner.pre_moderation';
+const workflowPost = 'components.preprint-status-banner.post_moderation';
+
+const iconPending = 'fa-hourglass-o';
+const iconAccepted = 'fa-check-circle-o';
+const iconRejected = 'fa-times-circle-o';
+
+export default Ember.Component.extend({
+    i18n: Ember.inject.service(),
+    theme: Ember.inject.service(),
+
+    // translations
+    labelModeratorFeedback: 'components.preprint-status-banner.feedback.moderator_feedback',
+    moderator: 'components.preprint-status-banner.feedback.moderator',
+    feedbackBaseMessage: 'components.preprint-status-banner.feedback.base',
+    baseMessage: 'components.preprint-status-banner.message.base',
+
+    classNames: ['preprint-status-component'],
+    classNameBindings: [
+        'isPendingPre:preprint-status-pending-pre:preprint-status-pending-post',
+        'isAccepted:preprint-status-accepted:',
+        'isRejected:preprint-status-rejected:',
+    ],
+    attributeBindings: ['hidden'],
+
+    hidden: computed('model.provider.reviewsWorkflow', function() {
+        return this.get('model.provider.reviewsWorkflow') === 'none';
+    }),
+
+    reviewerComment: '',
+    moderatorName: '',
+
+    init() {
```

Comment:
I may be thinking of controllers but IIRC `init` is not called every time `model` changes.
Have you switched back and forth between multiple preprints to make sure this updates correctly?